build/target-repository/docs/rector_rules_overview.md

```diff
@@ -2068,21 +2068,45 @@ Order attributes by desired names
 
 - class: [`Rector\CodingStyle\Rector\ClassMethod\OrderAttributesRector`](../rules/CodingStyle/Rector/ClassMethod/OrderAttributesRector.php)
 
+#### 1) Order by specific namespace
+
+```php
+use Rector\CodingStyle\Rector\ClassMethod\OrderAttributesRector;
+use Rector\Config\RectorConfig;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->ruleWithConfiguration(OrderAttributesRector::class, [Annotation\First::class, Annotation\Second::class]);
+};
+```
+
+↓
+
+```diff
++#[Annotation\First]
+ #[Annotation\Second]
+-#[Annotation\First]
+ class Someclass
+ {
+ }
+```
+
+#### 2) Order alphabetically
+
 ```php
 use Rector\CodingStyle\Rector\ClassMethod\OrderAttributesRector;
 use Rector\Config\RectorConfig;
 
 return static function (RectorConfig $rectorConfig): void {
-    $rectorConfig->ruleWithConfiguration(OrderAttributesRector::class, ['First', 'Second']);
+    $rectorConfig->ruleWithConfiguration(OrderAttributesRector::class, [Rector\CodingStyle\Rector\ClassMethod::ALPHABETICALLY]);
 };
 ```
 
 ↓
 
 ```diff
-+#[First]
- #[Second]
--#[First]
++#[Annotation\AAttribute]
+ #[Annotation\BAttribute]
+-#[Annotation\AAttribute]
  class Someclass
  {
  }
```

Comment:
Btw, this file is generated from `getRuleDefinition()` so these changes will be overriden. Any information that you want to see in the docs should be in `getRuleDefinition()`